docs/src/usage/client.rst

```diff
@@ -0,0 +1,41 @@
+********************************************
+Query submission via the MC\ :sup:`2` Client
+********************************************
+
+Starting Opaque SQL
+###################
+
+Here we outline how to use `the client <https://github.com/mc2-project/mc2>`_ to submit queries. In this security model, the Spark driver is considered *untrusted*, and should not have any way to decrypt results or any intermediate computation.
+
+**Note** that this method of computation is still not entirely supported. It is possible to submit queries from MC\ :sup:`2` Client, but only with a shared key on the driver, so it must still be considered *trusted*.
```

Comment:
This is a bit confusing, since for security reasons the driver still needs to be located on the client side. I would say that this is an alternative way to run Opaque SQL that integrates well with the rest of the mc2 projects, since the mc2-client also works with Secure XGBoost?